src/content/docs/developer/guides/automated-screenshot-management.mdx

```diff
@@ -0,0 +1,101 @@
+---
+title: Automating Screenshot Management
+description: Learn how to automate screenshot creation and updates with Crowdin In-Context and Playwright
+slug: developer/automated-screenshot-management
+---
+
+import { Steps } from '@astrojs/starlight/components';
+
+This guide explains how to automate generating and updating screenshots in Crowdin [In-Context](/developer/in-context-localization/).
+
+By automating screenshot creation, translators gain accurate and up-to-date visual context, improving the localization process and translation quality. The guide covers prerequisites, setup instructions, capturing screenshots across web pages, and uploading them to Crowdin via API.
+
+**Prerequisites**:
```

Comment:
heading

Reply:
This section looks complicated and overwhelming. I'd suggest:

- removing the bullet list
- "Verify that your website integrates Crowdin In-Context functionality" - we can mention this at the beginning of the Prerequisites section, e.g. `Before proceeding, make sure ...` + link to the integration section
- creating a sub-heading **Account Setup** with the steps component
- creating a sub-heading **Dependencies** that describes the Playwright and otplib dependencies

Reply:
> heading

The only other instance of "Prerequisites" is in the Quick Start article, where it's formatted the same way I did here. Should I update the formatting in the Quick Start article to match this (i.e., using a heading instead of bold text)?

Reply:
For the Quick Start article, that's fine, since it's just two simple points. The current case is different and more complex.